package-managers/nuget.md

```diff
@@ -1,25 +1,76 @@
 # NuGet \(.NET\)
-
 TODO (inspiration: http://docs.myget.org/docs/walkthrough/getting-started-with-nuget)
 
-## Connecting to a NuGet feed
 
+## Connecting to a NuGet feed
 public feed, private feed with basic auth, pre-authenticated feed URL
 
-### Credential Provider for Visual Studio
 
+### Find your feed URL
+
+#### Public feed
+There are two types of public feeds you can connect to. A public feed you own and one from the gallery. The only real difference here, is where to find the URL of the feed.
+
+With this URL you can connect to the feed using your favorite tool, which could be [Visual Studio](#visual-studio), [JetBrains Rider](#jetbrains-rider) or the [NuGet-CLI](#nugetexe)
+
+
+##### Gallery feed
+When connecting to a gallery feed you first click on the feed you want to connect to and there click on the "Connect to feed" button.
+
+![Gallery feed details](assets/gallery-feed.png)
+
+A dialog opens where you will once again find your 'NuGet V3' URL.
+
+![Gallery feed URL](assets/connect-gallery-feed.png)
+
+With this URL you can connect to the feed using your favorite tool, which could be [Visual Studio](#visual-studio), [JetBrains Rider](#jetbrains-rider) or the [NuGet-CLI](#nugetexe)
+
+##### Private feed
+To consume private feeds you need to get consume permissions from the feed owner or admin. When you have these rights you can navigate to the 'feed details' where you can find the 'NuGet V3' URL. 
+
+![Feed details](assets/feed-details.png)
+
+With this URL you can connect to the feed using your favorite tool, which could be [Visual Studio](#visual-studio), [JetBrains Rider](#jetbrains-rider) or the [NuGet-CLI](#nugetexe)

```

Comment:
This paragraph is a duplicate of what you wrote [here](https://github.com/maartenba/myget-documentation/pull/1/files#diff-a6e799b6b0d7c47c3631837ea0698fdeR14).

However, I'd add a comment here with anchors to quickly navigate to the client-specific **Private Feed** sections below.